File: Programmers/Lv.0/문자열 뒤집기/181905.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String answer = "";
        
        String[] str = my_string.split("");
        for (int i=0; i<=(e-s)/2; i++) {
            String temp = str[s + i];
            str[s + i] = str[e - i];
            str[e - i] = temp;
        }
        
        answer = String.join("", str);
        
        return answer;
    }
}

File: Programmers/Lv.0/문자열 뒤집기/181905.js
function solution(my_string, s, e) {
    let answer = '';
    
    const str = [...my_string];
    for (let i=0; i<=(e-s)/2; i++) {
        const temp = str[s + i];
        str[s + i] = str[e - i];
        str[e - i] = temp;
    }
    
    answer = str.join('');
    
    return answer;
}
